typing/typemod.ml

```diff
@@ -3414,6 +3415,14 @@ let report_error ~loc _env = function
         (Style.as_inline_code longident) lid
         Style.inline_code (Path.name path)
         Style.inline_code (Ident.name id)
+  | With_lost_alias(mod_id, lost_alias_path) ->
+      Location.errorf ~loc
+        "@[<v>\
+           @[This deep destructive %a substitution inside of %a would @ \
+             loose the aliasing to %a @].@]"
```

Comment:
loose ⇒ lose. Also it is better to avoid opening boxes when possible, and add break hints  before and after large inline block of code:
```ocaml
This deep destructive %a substitution inside@ of@ %a@ would @ \
lose the aliasing to %a.
```